### DISCUSSION TITLE
ZSH_AUTOSUGGEST_CLEAR_WIDGETS shows on clear command

### DISCUSSION THREAD

**Discussion by jsadowyj** [marlonrichert/zsh-autocomplete#464] [Q&A] [answered]
Hi everyone,

The fact that it tries autocompleting clear to ZSH_AUTOSUGGEST_CLEAR_WIDGETS= really bothers me. Is there any way I can turn this off?

<img width="1090" alt="image" src="https://user-images.githubusercontent.com/47227322/183544735-97e71e1c-7a54-41e0-9a50-a00337db9841.png">

**Comment by marlonrichert** [comment, marked as answer]
Normally, Autocomplete does not show any command completions that match the current word on the command line exactly.

To disable this, add to your `.zshrc` file _after_ sourcing Zsh Autocomplete:
```zsh
zstyle ':completion:*:-command-:*' ignore-line no
```

For those cases where you don't want this behavior, just press <kbd>Ctrl</kbd><kbd>Space</kbd> to reveal all possible completions.
